packages/embed-grid/src/App.tsx

```diff
@@ -103,54 +103,63 @@ function App(): JSX.Element {
     setIsLoading(false);
   }, [searchParams]);
 
-  useEffect(() => {
-    initApp();
-  }, [initApp]);
-
-  useEffect(() => {
-    function receiveMessage(e: MessageEvent) {
-      const { command, value } = e.data as { command: Command; value: unknown };
-      switch (command) {
-        case 'filter': {
-          const filterCommandValues = value as FilterCommandType[];
-          const newInputFilters = filterCommandValues.map(
-            ({ name, value: filterValue }) => {
+  useEffect(
+    function initializeApp() {
+      initApp();
+    },
+    [initApp]
+  );
+
+  useEffect(
+    function handleTableCommand() {
```

Comment:
This outer function is for registering the listeners, not for handling the table command itself. A more appropriate name would be `initCommandListener`:
```suggestion
    function initCommandListener() {
```